From 141ac41dd5d850160bd02a9ddcb27052e0df663d Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 13 Oct 2014 22:29:59 -0400
Subject: [misc] genwqe: Fix compile problems for Alpha

Message-id: <1413239416-7778-12-git-send-email-gduarte@redhat.com>
Patchwork-id: 97542
O-Subject: [RHEL7.1 PATCH BZ 1088491 11/28 v2] GenWQE: Fix compile problems for Alpha
Bugzilla: 1088491
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088491
Upstream Status: 90b4e97e69ff3d6cca3e227145c7de8305bd84cf

commit 90b4e97e69ff3d6cca3e227145c7de8305bd84cf
Author: Frank Haverkamp <haver@linux.vnet.ibm.com>
Date:   Tue Jan 7 15:41:24 2014 +0100

    GenWQE: Fix compile problems for Alpha

    The header which contained the declaration for kcalloc() was not
    inlcuded.

    Reported-by: kbuild test robot <fengguang.wu@intel.com>
    Signed-off-by: Frank Haverkamp <haver@linux.vnet.ibm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/genwqe/card_base.h b/drivers/misc/genwqe/card_base.h
index 41953e3..5e4dbd2 100644
--- a/drivers/misc/genwqe/card_base.h
+++ b/drivers/misc/genwqe/card_base.h
@@ -36,6 +36,7 @@
 #include <linux/io.h>
 #include <linux/version.h>
 #include <linux/debugfs.h>
+#include <linux/slab.h>
 
 #include <linux/genwqe/genwqe_card.h>
 #include "genwqe_driver.h"
diff --git a/drivers/misc/genwqe/card_ddcb.c b/drivers/misc/genwqe/card_ddcb.c
index b5a3206..6f1acc0 100644
--- a/drivers/misc/genwqe/card_ddcb.c
+++ b/drivers/misc/genwqe/card_ddcb.c
@@ -38,6 +38,7 @@
 #include <linux/interrupt.h>
 #include <linux/crc-itu-t.h>
 
+#include "card_base.h"
 #include "card_ddcb.h"
 
 /*
-- 
1.7.1